target/consensource-ui
# refactor: remove go back button

Signed-off-by: Patrick-Erichsen <Patrick.Erichsen@target.com>

## Proposed change/fix

- Removes the `<HistoryGoBackButton />`. @trevormcdonald and I decided that this was unnecessary in the current state of the app, and was introducing unnecessary complexity.
- Renames `<LinkWithHistory />` to `<UnstyledLink />` to reflect that it no longer adds state to the history object.

## Types of changes

What types of changes does this pull request introduce to ConsenSource? _Put an `x` in the boxes that apply_


## How to run/test

- `yarn start` and confirm no errors are emitted
- Visit a factory profile page and confirm that the header is formatted correctly
- Visit a factory certification page and confirm that the header is formatted correctly
